code/street-banner/street-banner.css

```diff
@@ -42,3 +42,22 @@ a.big-button-container {
 a.small-button {
   text-decoration: none;
 }
+
+.group-layout-wrapper .view-column {
+  display: block;
+}
```

Comment:
you should create a CSS class for these since they are the same styling and you can apply it to other views in the future as well without worrying about view #'s

you can also access an entire `a` class with:
`a.whatever {}`

Knack also likes overriding div styles so you also want to be cognizant if you need to use the `!important` tag. I overwrite Knack Detail Labels with `#view_1278 div.kn-detail-label { background-color: transparent; min-width: 0% !important; }` for example so that the Knack theme even if updated does not change the styling.

Reply:
We synced and found out that Chrispin's suggestion only applies to "trigger links" and I was using a menu button.